wdk-ddi-src/content/vpci/nc-vpci-vpci_write_block.md

```diff
@@ -0,0 +1,94 @@
+---
+UID: NC:vpci.VPCI_WRITE_BLOCK
+tech.root: pci
+title: VPCI_WRITE_BLOCK
+ms.date: 09/29/2022
+targetos: Windows
+description: Learn more about the VPCI_WRITE_BLOCK routine
+prerelease: false
+req.assembly: 
+req.construct-type: function
+req.ddi-compliance: 
+req.dll: 
+req.header: vpci.h
+req.idl: 
+req.include-header: 
+req.irql: DISPATCH_LEVEL
+req.kmdf-ver: 
+req.lib: 
+req.max-support: 
+req.namespace: 
+req.redist: 
+req.target-min-winverclnt: 
+req.target-min-winversvr: Windows Server 2012
+req.target-type: 
+req.type-library: 
+req.umdf-ver: 
+req.unicode-ansi: 
+topic_type:
+ - apiref
+api_type:
+ - UserDefined
+api_location:
+ - vpci.h
+api_name:
+ - VPCI_WRITE_BLOCK
+f1_keywords:
+ - VPCI_WRITE_BLOCK
+ - vpci/VPCI_WRITE_BLOCK
+dev_langs:
+ - c++
+helpviewer_keywords:
+ - VPCI_WRITE_BLOCK
+---
+
+## -description
+
+The *WriteVfConfigBlock* routine writes a block of configuration data for a PCI Express virtual function (VF). This routine is called by the driver of a PCIe VF on a device that supports the single root I/O virtualization (SR-IOV) interface.
+
+## -parameters
+
+### -param Context \[in\]
+
+A pointer to interface-specific context information. The caller passes the value that is passed as the **Context** member of the [**VPCI\_INTERFACE\_STANDARD**](/windows-hardware/drivers/ddi/vpci/ns-vpci-vpci_interface_standard) structure for the interface.
```

Comment:
I'm pretty sure that Meriwether will complain if SAL is on the ### -param line. Change all of these to be formatted as follows:

### -param Context

[in] A pointer to ...